.github/workflows/ci.yml

```diff
@@ -62,6 +62,10 @@ jobs:
       if: steps.cache-ext.outputs.cache-hit != 'true'
       working-directory: ./ext
       run: bash libjpeg.cmd
+    - name: Build libsharpyuv
+      if: steps.cache-ext.outputs.cache-hit != 'true'
+      working-directory: ./ext
+      run: bash libsharpyuv.cmd
```

Comment:
Also add this step to the new file .github/workflows/ci-windows.yml. Change the last line to:

```
      run: ./libsharpyuv.cmd
```